Server/src/genericInterfaces/ResponsesInterface.ts

```diff
@@ -0,0 +1,19 @@
+export interface IResponse {
+    status: string
+    statusCode: number
+    response?: string
+}
+
+export interface SuccessIResponse extends IResponse {
+    status: "Success",
+    statusCode: 200,
+    adminRights: string;
+}
+
+export interface FailureIResponse extends IResponse {
```

Comment:
do we need Failure Response, Forbidden response if we are throwing HTTP exceptions? i think the exception should handle it maybe, either way the statusCode, status and message should come from the exception

Reply:
Above